Geometry/HcalCommonData/src/HcalNumberingFromDDD.cc

```diff
@@ -5,11 +5,10 @@
 
 #include "CondFormats/GeometryObjects/interface/HcalParameters.h"
 #include "Geometry/HcalCommonData/interface/HcalNumberingFromDDD.h"
+#include "DetectorDescription/DDCMS/interface/DDUnits.h"
```

Comment:
```suggestion
#include "DetectorDescription/Core/interface/DDUnits.h"
```

Reply:
There are two versions of `DDUnits.h`. They will be consolidated and moved somewhere more central soon.

Reply:
Done